addon/components/paper-tab.js

```diff
@@ -57,7 +57,9 @@ export default Component.extend(ChildMixin, RippleMixin, FocusableMixin, {
   },
 
   click() {
-    this.sendAction('onClick', ...arguments);
-    this.sendAction('onSelect', this);
+    if(!this.get("disabled")) {
```

Comment:
@MarcoUmpierrez could you please change this line to `if (!this.get('disabled')) {` to conform with the rest of the project's coding style?